README.md

```diff
@@ -68,3 +81,53 @@ MATMUL          5.49    1.04    5.28x   81.07%
 NPB_CG          34.99   10.21   3.43x   70.82%
 PI              3.33    0.46    7.30x   86.30%
 ```
+
+### Real test cases
+
+```
+$ ./benchmark-mbedTLS-vector.sh
+```
+```
+...
+##################################################
+7/7. Verifying the speedup of the vectorized version ...
+##################################################
+Press enter to continue
+
+
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+aes_xts algorithm
```

Comment:
Another algorithm that experiences speed improvements with this is AES-CBC. @sergioVaz00, please confirm!

Reply:
Yes, even if only one of the loops is vectorized, a small speed improvement is obtained:
original:
  ARIA-CBC-128             :     156038 KiB/s,         18 cycles/byte
  ARIA-CBC-192             :     137425 KiB/s,         20 cycles/byte
  ARIA-CBC-256             :     122271 KiB/s,         22 cycles/byte

vectorized:
  ARIA-CBC-128             :     165943 KiB/s,         17 cycles/byte
  ARIA-CBC-192             :     145935 KiB/s,         19 cycles/byte
  ARIA-CBC-256             :     129778 KiB/s,         21 cycles/byte

Do you want me to add it so you can make the new benchmark?